src/components/EventForm/EventForm.tsx

```diff
@@ -99,7 +99,7 @@ function EventForm({ onSubmit, ...rest }: EventFormProps) {
     <Formik
       initialValues={initialValues}
       validationSchema={EventFormSchema}
-      onSubmit={async (values, { setSubmitting }) => {
+      onSubmit={ async (values, { setSubmitting }) => {
```

Comment:
Unnecessary spacing. You can configure Prettier to be automatically be ran after each save in your editor